src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -551,10 +547,9 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
                 percentageFrequencyOrAverage[i][fqd.numOfRubricChoices] += choiceWeight;
             }
         }
-        
         return percentageFrequencyOrAverage;
```

Comment:
To follow SLAP more, the second block (from lines 535-550) should also be extracted to another private method.